docs-src/cloud/api-keys/manage-api-keys.md

```diff
@@ -0,0 +1,122 @@
+---
+id: manage-api-keys
+title: Manage API Keys
+sidebar_label: Manage API Keys
+description: Manage API Keys by creating, deleting, or updating access to them using the Cloud UI or tcld.
+tags:
+  - how to
+---
+
+Manage API Keys by creating, deleting, or updating access to them using the Cloud UI or tcld.
+
+### Generate an API Key
+
+:::note
+
+Once generated, copy and securely save the API Key.
+It will be displayed only once for security purposes.
+
+:::
+
+Generate an API Key using one of the following methods.
+
+#### Using the Cloud UI
+
+To generate an API Key using the Cloud UI:
```

Comment:
The instructions here are for Global Admins.  May we change these instructions to follow the user How to guidance provided [here](https://www.notion.so/temporalio/Docs-for-API-Keys-78085708c5d64edd894dbb69ea9b82a5?pvs=4#0a3113df5e5a40268e46b896bad4a5ef).  For context, only Global Admins can access the Settings section of Temporal Cloud. 

To generate an API Key using the Cloud UI:

1. [[Login](https://cloud.temporal.io/)](https://cloud.temporal.io/) to the Cloud UI
2. Navigate to [[Profile Page → API Keys](https://cloud.temporal.io/profile/api-keys)](https://cloud.temporal.io/profile/api-keys)
3. Click the Create API Key button
4. Provide a Name, an optional description, and an Expiration period
5. Click Generate API Key